.tito/lib/origin/builder/__init__.py

```diff
@@ -55,15 +55,37 @@ def _setup_test_specfile(self):
             )
             # Custom Openshift v3 stuff follows, everything above is the standard
             # builder
-            cmd = '. ./hack/common.sh ; echo $(os::build::ldflags)'
+            
+            ## Fixup os_git_vars
```

Comment:
The big difference is that os::build::ldflags echo's everything out in one line.  os::build::os_version_vars just set's variables.  But I'll give it a try.